frontend/src/lib/i18n/locales/en.json

```diff
@@ -14,6 +14,18 @@
     "project_table_title": "Projects",
     "user_table_title": "Users"
   },
+
+  "account_settings": {
+    "title": "Account Settings",
+    "label_email": "change email (unavailable for now)",
+    "label_username": "change username (unavailable)",
```

Comment:
Let's just remove this line. We can add it back in later once we add the username to the user data stored in Svelte, at which time we'll add the field to the form and put the label text back in here.